pkg/core/transformer/main.go

```diff
@@ -100,31 +100,50 @@ func (t *Transformer) Apply(input string) (output string, err error) {
 
 		case "findSubMatch":
 
-			val, ok := value.(string)
-
-			if !ok {
-				return "", fmt.Errorf("unknown value for findSubmatch: %v", val)
+			f := FindSubMatch{}
+
+			// Here I deal with 2 ways of providing the data within the yaml
+			// only one parameter in string which is the pattern
+			// considering 0 as the CaptureIndex default
+			// or receiving both parameter as a map.
```

Comment:
```suggestion
			// If the manifest value is only a string (the pattern), then 0 is the implied captureIndex value
			// Otherwise, both pattern and captureIndex are retrieved from the map value of the manifest
```